src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const LONG_TERM = 7;
+  const LONG_TERM_DISKOUNT = 50;
```

Comment:
There is a typo in the variable name LONG_TERM_DISKOUNT. It should be LONG_TERM_DISCOUNT.